api/controllers/game/subscribe.js

```diff
@@ -15,6 +15,12 @@ module.exports = function (req, res) {
     .then(async function success(arr) {
       // Catch promise values
       const [game, user] = arr;
+
+      // Fast fail if game is full
+      const gameIsFull = !game.status || game.players.length >= 2 || game.log.length > 0;
```

Comment:
I feel like this should be moved to a helper function if you’ll need it in more than one place ever. It could take game as an argument.

Reply:
Pulled into a new synchronous `isGameFull(game)` helper